api/services/profile.js

```diff
@@ -0,0 +1,69 @@
+const db = require('../../database');
+const errors = require('./errorHandlers/index');
+const jwt_decode = require('jwt-decode');
+
+const getProfile = async token => {
+  const { id } = jwt_decode(token);
```

Comment:
if future we need to write middleware for token decoding and we are using `jsonwebtoken` library for working with jwt tokens